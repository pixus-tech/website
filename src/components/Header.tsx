import { Link } from 'gatsby'
import React from 'react'
import DarkLogo from 'images/wbm.svg'
import BrightLogo from 'images/wbm--bright.svg'

interface IProps {
  siteTitle: string
}

function Header({ siteTitle: _siteTitle }: IProps) {
  return (
    <header>
      <div className={'container'}>
        <div className={'top-menu'}>
          <div className={'logo'}>
            <Link to="/" title={'pixus'}>
              <DarkLogo className="darkLogo" />
              <BrightLogo className="brightLogo" />
            </Link>
          </div>

          <div className={'get-started'}>
            <a href={'https://app.pixus.tech'}>open pixus</a>
          </div>
        </div>
      </div>
    </header>
  )
}

export default Header
