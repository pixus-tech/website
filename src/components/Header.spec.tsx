import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

describe('Header', () => {
  it('displays the corect cta', () => {
    const { queryByText } = render(<Header siteTitle="pixus" />)

    expect(queryByText('open pixus')).not.toBeNull()
  })
})
