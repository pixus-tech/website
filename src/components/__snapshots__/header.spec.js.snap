// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="container"
  >
    <div
      className="top-menu"
    >
      <div
        className="logo"
      >
        <a
          href="/"
          title="pixus"
        >
          <img
            alt="pixus logo"
            className="darkLogo"
            src="test-file-stub"
          />
          <img
            alt="pixus logo"
            className="brightLogo"
            src="test-file-stub"
          />
        </a>
      </div>
      <div
        className="get-started"
      >
        <a
          href="https://app.histaff.io"
        >
          Get Started
        </a>
      </div>
    </div>
  </div>
</header>
`;
